tests/_support/Step/Acceptance/Administrator/Content.php

```diff
@@ -42,7 +42,7 @@ public function iSaveAnArticle()
 	public function iShouldSeeTheMessage($message)
 	{
 		$I = $this;
-		$I->waitForText($message, 60, AdminPage::$systemMessageContainer);
```

Comment:
I have tried to find all the other files for timout, I think this is the only place I found so far. I will add and use `TIMEOUT` constant to replace all the static time. Thank you.